### DISCUSSION TITLE
Does Driver helper plan to offer only Asus provided drivers or also updated options from manufacturer?

### DISCUSSION THREAD

**Discussion by jaycc3000** [seerge/g-helper#695] [General]
Driver Pulled by G-Helper:
Asus Version V10.031.0828.2018 (Dated 2020 by Asus on their download page)

- https://www.realtek.com/en/downloads
-- Download PCIe: Install_PCIE_Win11_11.13.0424.2023_05052023
-- Driver Version listed in Windows Device Manager: 1168.13.424.2023

**Comment by seerge** [comment]
@jaycc3000 app checks for available drivers only via asus api (same as myasus)

**Comment by jaycc3000** [comment]
That's fair. Helping people get things working after a fresh installation is helpful :-)

I don't know how long Asus will keep downloads available via their API, but I would hope for a long while :-)

**Reply by seerge** [reply]
I think its forever :) they usually stop updating gpu drivers after some time, but for that its better to go to amd/nvidia sites anyway
